frontend/css/layout.css

```diff
@@ -232,3 +248,99 @@ li a {
         grid-template-columns: 1fr;
     }
 }
+
+.cart {
```

Comment:
Could all the styles that are specific for the cart be placed in a separate file?

Reply:
new file for cart has been created